### PR TITLE
docs(FAQ): move to catppuccin namespace

### PR DESCRIPTION
**Pull request by 42willow**
* https://github.com/catppuccin/nix/pull/371

oh and we need a port for NüschtOS Search :)

### CODE REVIEW

File: docs/src/faq.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # FAQ
 
 - Q: **"How do I know what programs are supported?"**\
-  A: You can find programs supported through home-manager [here](options/home-manager-options.md),
-  and NixOS modules [here](options/nixos-options.md)
+  A: You can find programs supported through home-manager [here](search/rolling/?scope=home-manager+modules),
```
Comment (isabelroses) [inline docs/src/faq.md:4]:
```suggestion
  A: You can find programs supported through home-manager [here](https://nix.catppuccin.com/search/rolling/?scope=home-manager+modules),
```

File: docs/src/faq.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  and NixOS modules [here](search/rolling/?scope=NixOS+modules)
```
Comment (isabelroses) [inline docs/src/faq.md:5]:
```suggestion
  and NixOS modules [here](https://nix.catppuccin.com/search/rolling/?scope=NixOS+modules)
```

File: docs/src/faq.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 - Q: **"How do I set `catppuccin.enable` for everything I use?"**\
-  A: You can set `catppuccin.enable` [globally](options/nixos-options.md#catppuccinenable)
+  A: You can set `catppuccin.enable` [globally](search/rolling/?option_scope=0&option=catppuccin.enable)
```
Comment (isabelroses) [inline docs/src/faq.md:8]:
```suggestion
  A: You can set `catppuccin.enable` [globally](https://nix.catppuccin.com/search/rolling/?option_scope=0&option=catppuccin.enable)
```

**Review by isabelroses — CHANGES_REQUESTED** [review]
I think we should use full links rather then relative links, such that people using github to view the files, or people who have the repo cloned know where to find things.

**Review by getchoo — APPROVED** [review]
Thanks!

**Pull request merged**